Makefile

```diff
@@ -39,17 +36,28 @@ coverage: ## Run pytest with coverage report.
 	coverage report -m
 	coverage html
 
+.PHONY: preparedocs
+preparedocs: ## move file from the sphinx-starter-pack to docs folder
+	git submodule update --init -- sphinx-starter-pack
+	cp sphinx-starter-pack/.sphinx/_static/* docs/_static
+	mkdir -p docs/_templates
+	cp sphinx-starter-pack/.sphinx/_templates/* docs/_templates
```

Comment:
I think you can just copy the entire folder, recursively...so no need to `mkdir`